src/picker.js

```diff
@@ -104,13 +104,24 @@ function init({tabs: originalTabs, env}) {
 			}
 			const counter = document.createElement("div");
 			counter.className = "tab-image-counter";
+			counter.dataset.i18n = "pickerGroupCounterClicked|title";
+			counter.onclick = (e) => {
```

Comment:
We don't have such "clickable span" in the extension. I think it will be better to construct a real button to do the work.

How about replacing div with a button? It will work like the `Invert selection` button and look like the `Save (xx/yy)` button:
```html
<li class="image-container">
  ... labels ...
  <button class="invert tab-image-counter">
    Invert
  </button>
</li>
```
The counter is CSS based so you shouldn't have to worry about breaking it if the class name is still there. Though it might be better to put the counter after the button text i.e. `::before -> ::after`.

Reply:
A button is nice for UX semantics, but I'm concerned it might be too eye-catching especially on certain platform themes, and I'm not good at customizing a stylized button. A check box would be better but need to change its state based on the situation.